lib/rules/no-debug.ts

```diff
@@ -13,7 +13,7 @@ import { ASTUtils, TSESTree } from '@typescript-eslint/experimental-utils';
 
 export const RULE_NAME = 'no-debug';
 export type MessageIds = 'noDebug';
-type Options = [];
+type Options = [{ utilNames: string[] }];
```

Comment:
I was under the impression there were more functions that users might like to ignore and that you'd not want to maintain a list of all them in the code; given that you do, I think it might be nicer to use an object rather than array, since the latter gives the impression (imo) that it accepts arbitrary strings.

i.e:

```
interface DebugUtilsToCheckFor {
  debug?: boolean;
  logTestingPlaygroundURL?: boolean;
}

type Options = [{ utilsToCheckFor: DebugUtilsToCheckFor }]
```

It probably has a more sensible default behaviour too, since with an array as soon as you specify that option all the defaults go out the door unless the user maintains the list themselves; whereas with an object, users can toggle specific utils without having to care about the others.

(all of this a bit moot when it's just two names, but tbh if more exist currently we might as well include them all right now).

I'm not too fussed personally, but just know from experience how much of a pain these sorts of things can be to change afterwards 😅

Reply:
To be honest I'm happy with this implementation too, just wanted to type possibilities for the final user. Although I prefer the list in case we need to include more debugging names (which might happen to cover dom-testing-library ones).

Reply:
I'm not sure I follow you - in both cases you have the types for the final user and it's equal amount of work to add more debugging names.

But I'm happy to go with whichever you want :)

Reply:
You were right: defining the utils to be checked as an object is way better than as an array.